cime/src/drivers/moab/main/prep_wav_mod.F90

```diff
@@ -0,0 +1,361 @@
+module prep_wav_mod
+
+  use shr_kind_mod    , only: r8 => SHR_KIND_R8
+  use shr_kind_mod    , only: cs => SHR_KIND_CS
+  use shr_kind_mod    , only: cl => SHR_KIND_CL
+  use shr_sys_mod     , only: shr_sys_abort, shr_sys_flush
+  use seq_comm_mct    , only: num_inst_atm, num_inst_ice, num_inst_ocn
+  use seq_comm_mct    , only: num_inst_wav, num_inst_frc
+  use seq_comm_mct    , only: CPLID, WAVID, logunit
```

Comment:
Again, this seems like far too much copy/paste to be sustainable.

Reply:
I'm not sure what the Fortran files are going to look like when we're done. For now I'll use symlinks for files that are currently unchanged, and look for a better way to engineer this